tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -152,7 +152,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
         if (!functions_worker_runtime_value.empty() && !IsAzureFunctionsEnabled())
         {
             Logger::Info("DATADOG TRACER DIAGNOSTICS - Profiler explicitly disabled for Azure Functions.");
```

Comment:
@pierotibou: I thought auto-instrumentation worked in Azure Functions. Is this check wrong?

Reply:
The `IsAzureFunctionsEnabled()` function basically does looks for the `DD_TRACE_AZURE_FUNCTIONS_ENABLED` environment variable and returns `true` if not set, so this is fine

Reply:
My mistake. I read `IsAzureFunctionsEnabled()` as "are we running in Azure Functions?", similar to the check for Kudu above 🤦🏽‍♂️